packages/medusa/src/commands/develop.js

```diff
@@ -19,10 +18,8 @@ export default async function ({ port, directory }) {
   })
 
   const cliPath = path.join(directory, "node_modules", ".bin", "medusa")
-  let child = spawn(cliPath, [`start`, ...args], {
-    cwd: directory,
-    env: process.env,
-    stdio: ["pipe", process.stdout, process.stderr],
+  let child = fork(cliPath, [`start`, ...args], {
```

Comment:
Suggestion: we should probably pass the detach option as well as call the unref on the child

Reply:
If we pass the detached option then I appear to lose stdio from the Medusa `start` command. I tested it this way, trying a few values for the `stdio` and `silent` options, but the only output I get is from babel.

Reply:
Could you try to set the silent option to true, apparently it should pipe stdin and out to the parent

Reply:
I tried that, didn't appear to work on my machine 🤔 I also found the same suggestion online I think

Reply:
Humm, so let’s wait a bot so ce there is a workaround at the moment. And i ll look into it later 👍 is that alright ?

Reply:
Of course 👍🏻

Reply:
Really noce that you figured it out 🤘 i wasn’t sure about the detach, just that they does not required biderectional coms 😂

Reply:
Are you sure a out the stdio defaults? Could you look at the fork impl just to be sure

Reply:
[Just linking here for clarity](https://github.com/nodejs/node/blob/v16.x/lib/child_process.js#L156)

Reply:
I dont think we want to make it silent then

Reply:
Is the current solution OK, or shall I be explicit about inheriting stdio by passing the option?

Reply:
I think that it is fine like this